server/repository/src/migrations/v1_06_00/master_list_line_add_item_link_id.rs

```diff
@@ -0,0 +1,41 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    if cfg!(feature = "postgres") {
+        sql!(
+            connection,
+            r#"
+            ALTER TABLE master_list_line 
+            ADD COLUMN item_link_id TEXT NOT NULL DEFAULT 'temp for migration';
+            
+            UPDATE master_list_line SET item_link_id = item_id;
+            
+            ALTER TABLE master_list_line 
+            ADD CONSTRAINT master_list_line_item_link_id_fkey 
+            FOREIGN KEY (item_link_id) REFERENCES item_link(id);
+            DROP INDEX IF EXISTS index_master_list_line_item_id_fkey;
+            ALTER TABLE master_list_line DROP item_id;
+            "#,
+        )?;
+    } else {
+        sql!(
+            connection,
+            r#"
+            ALTER TABLE master_list_line RENAME TO master_list_line_old;
+
+            CREATE TABLE master_list_line (
```

Comment:
Hmm we were generally trying to avoid the ol' recreating tables trick, though `master_list_line` shouldn't be so bad as say `invoice_line`.

No luck using the OG PR's `PRAGMA foreign_keys = OFF;`?

Reply:
The FK constraint was done a bit differently for this table...`FOREIGN KEY(item_id) REFERENCES item(id)`, so can't drop the FK constraint with `PRAGMA foreign_keys = OFF`.... 

SQLite doesn't allow you to drop foreign key constraints unless you `DROP` the table..